docs/antora.yml

```diff
@@ -0,0 +1,20 @@
+name: maven-plugin
+title: Asciidoctor Maven Plugin
```

Comment:
I can't decide whether we should drop "Asciidoctor" from the component title. It seems redundant to say "Asciidoctor Maven Plugin" on the Asciidoctor docs site. WDYT?

Reply:
Is just for the title at the top of the toc. If that's not going be displayed in the final site, I image I can remove it from the UI theme I am using. wdyt?
![image](https://user-images.githubusercontent.com/5781153/100021391-b1465480-2de1-11eb-9ac0-6f665418b3e3.png)


Also, technically speaking we should call the project "Maven Tools". The Maven plugin and the doxia site module are bundled together for historic reasons but they are two separated components in truth.

Reply:
Seems very reasonable to name the docs that way using an umbrella term.